src/modules/menu/components/advancedTools/ImportFile/ImportFileButtons.vue

```diff
@@ -0,0 +1,74 @@
+<script setup>
+import { computed, toRef } from 'vue'
+import { useStore } from 'vuex'
+import { useI18n } from 'vue-i18n'
+
+const i18n = useI18n()
+const store = useStore()
+const emit = defineEmits(['loadFile'])
+
+// Props
+const props = defineProps({
+    buttonState: {
+        type: String,
+        default: 'default',
+    },
+    disabled: {
+        type: Boolean,
+        default: false,
+    },
+})
+
+// Reactive data
+const buttonState = toRef(props, 'buttonState')
+
+// Store mapping (input)
+const toggleImportFile = () => store.dispatch('toggleImportFile')
+
+// Computed properties
+const buttonI18nKey = computed(() => {
+    switch (buttonState.value) {
+        case 'loading':
+            return 'loading_file'
+        case 'succeeded':
+            return 'import_file_succeeded'
+        case 'default':
+        default:
+            return 'import'
+    }
+})
+const isLoading = computed(() => buttonState.value === 'loading')
+</script>
+
+<template>
+    <div class="mt-2 d-grid gap-2 d-md-flex justify-content-md-center">
```

Comment:
Regarding the use of `mt-2` here : as a good rule when building components : It's better to have all the positioning and margin conundrum at the parent level (which is the one placing the component side by side or in a container) and let the component itself just declare whatever it wants to show in the UI.

This way, if it is then re-used somewhere else, there's no need to cancel some margin or padding added here, it's a "blank canvas" that can be shaped however you want in each places it's called/used